packaging/aznfs/DEBIAN/control

```diff
@@ -6,4 +6,4 @@ Origin: Microsoft
 Maintainer: Azure Storage XNFSv3 IDC Team <xnfsv3devsidc@microsoft.com>
```

Comment:
Since this is a shared component now, should we switch to a different contact email, or just have all the maintainers join this one (the project won't be specific to v3 and IDC).